plugin/src/main/java/com/spotify/plugin/dockerfile/BuildMojo.java

```diff
@@ -93,7 +103,7 @@ public void execute(DockerClient dockerClient)
     }
 
     final String imageId = buildImage(
-        dockerClient, log, verbose, contextDirectory, repository, tag, pullNewerImage, noCache);
+        dockerClient, log, verbose, contextDirectory, repository, tag, pullNewerImage, noCache, buildArgs);
```

Comment:
Sounds like a good idea. If you want I could give it a go, unless you think it's outside the PR scope.